lambda/app/src/utils/constants.ts

```diff
@@ -1,2 +1,15 @@
 export const MS_GRAPH_URL = 'https://graph.microsoft.com';
 export const CYPRESS_MOCKED_IDIR_LOOKUP = [{ mail: 'pathfinder.ssotraining2@gov.bc.ca', id: 1 }];
+export const bcscIdpMappers = [
+  { name: 'username', type: 'oidc-username-idp-mapper', template: '${CLAIM.sub}@${ALIAS}' },
+];
+export const bcscClientScopeMappers = [
+  'given_name',
+  'email',
+  'display_name',
+  'given_names',
+  'family_name',
+  'sector_identifier_uri',
+  'address',
+];
+export const bcscDefaultScopes = 'openid profile email address';
```

Comment:
@jlangy Not all clients may be allowed to use address scope. Only if they select address specific attributes, we can add that scope to the IDP config. Any thoughts?

Reply:
yes, reading the spec it seems like they are using the claims as the fine grained control so I figured we can use their default scopes list. But I can add some logic to pare down the scopes too